DEMO_TUTORIAL.md

```
@@ -452,12 +452,12 @@ Antes de grabar, asegúrate de:
 
-## 🔗 Links Útiles
+##  Links Útiles
 
 - **Repositorio**: https://github.com/gilbertsahumada/movehat
 - **Movement Docs**: https://docs.movementnetwork.xyz
 - **Contrato Original**: https://docs.movementnetwork.xyz/devs/firstMoveContract
 
 ---
 
-**¡Buena suerte con tu demo! 🚀**
+**¡Buena suerte con tu demo! **
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)